decidim-initiatives/app/services/decidim/initiatives/initiative_search.rb

```diff
@@ -65,7 +64,7 @@ def search_type_id
 
       def search_author
         if author == "myself" && options[:current_user]
-          query.where(decidim_author_id: options[:current_user].id)
+          Decidim::Initiative.where(decidim_author_id: options[:current_user].id)
```

Comment:
Since Initiatives use the `Authorable` module their author could be polymorphic (although I don't think it can be done from the UI). Just to be safe, I'd also add:

```suggestion
          Decidim::Initiative.where(decidim_author_id: options[:current_user].id, decidim_author_type: Decidim::UserBaseEntity.name)
```

Reply:
Can we just use `Decidim::Initiative.where(decidim_author: options[:current_user])`? That should use the correct `type`, right? What do you think @marinavega @oriolgual ?

Reply:
Fixed! TY folks 😄